src/containers/Landing.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+
+import Gallery from '@containers/Gallery';
+import Loading from '@containers/Loading';
+import Splash from '@containers/Splash';
+import Store, {empty, loading} from '@models/Store';
+import { observer } from 'mobx-react';
+
+const Landing = observer((props) => {
+  // If the status is empty, attempt to load the user into the store.
+  useEffect(() => {
```

Comment:
This is going to be the "home page" if there is a user logged, then we just take them straight to the gallery, otherwise we show them the splash so they can log in. 

There maybe a small gap where the user is not loaded yet and we are waiting on the request. If that's the case, the we show them the loading screen.